src/sentry/api/endpoints/project_backfill_similar_issues_embeddings_records.py

```diff
@@ -22,36 +22,53 @@ class ProjectBackfillSimilarIssuesEmbeddingsRecords(ProjectEndpoint):
     }
 
     def post(self, request: Request, project: Project) -> Response:
+        # needs to have the flag to run
         if not features.has("projects:similarity-embeddings-backfill", project):
             return Response(status=404)
 
-        # needs to have the flag to run
-
+        # needs to either be a superuser or be in single org mode
         if not (is_active_superuser(request) or settings.SENTRY_SINGLE_ORGANIZATION):
             return Response(status=404)
 
-        # needs to either be a superuser or be in single org mode
+        # Define allowed keys
+        allowed_keys = {
+            "last_processed_id",
+            "skip_project_ids",
+            "only_delete",
+            "enable_ingestion",
+            "process_skipped_projects",
+        }
 
-        last_processed_id = None
-        only_delete = False
-        enable_ingestion = False
-        skip_processed_projects = False
-        skip_project_ids = None
+        # Check for any disallowed keys
+        disallowed_keys = set(request.data.keys()) - allowed_keys
+        if disallowed_keys:
+            return Response(
+                data={"detail": f"Disallowed keys: {', '.join(disallowed_keys)}"},
+                status=400,
+            )
 
-        if request.data.get("last_processed_id"):
-            last_processed_id = int(request.data["last_processed_id"])
-
-        if request.data.get("only_delete"):
-            only_delete = True
-
-        if request.data.get("enable_ingestion"):
-            enable_ingestion = True
+        last_processed_id = (
+            int(request.data["last_processed_id"])
+            if request.data.get("last_processed_id")
+            else None
+        )
+        skip_project_ids = (
+            request.data["skip_project_ids"].split(",")
+            if request.data.get("skip_project_ids")
+            else None
+        )
 
-        if request.data.get("skip_processed_projects"):
-            skip_processed_projects = True
+        # These overwrite the defaults of backfill_seer_grouping_records_for_project
+        only_delete = True if request.data.get("only_delete") else False
+        enable_ingestion = True if request.data.get("enable_ingestion") else False
+        # Notice that it reads as "process" rather than "skip", thus, it's the opposite of what you'd expect
+        skip_processed_projects = True if request.data.get("process_skipped_projects") else False
```

Comment:
This removes remembering to specify "skip_processed_projects" when starting a backfill.